Scripts/Linux/BuildGVFSForLinux.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+CONFIGURATION=$1
```

Comment:
Not needed for this PR but at some point you'll probably want this script to take a version number like the Mac build script does so that the version number of the build can be passed through.

Reply:
Done in 358254ec.